docs/example-scenario/aks-agic/aks-agic-content.md

```diff
@@ -122,6 +122,11 @@ Once deployed with the ability to observe multiple namespaces, AGIC will do the
 - Compose combined [Application Gateway config](https://github.com/Azure/azure-sdk-for-go/blob/37f3f4162dfce955ef5225ead57216cf8c1b2c70/services/network/mgmt/2016-06-01/network/models.go#L1710-L1744)
 - Apply the config to the associated Application Gateway via [ARM](/azure/azure-resource-manager/management/overview)
 
+> [!IMPORTANT]
+> You should not change the settings of Application Gateway directly. Every configuration change on Application Gateway should be performed from Ingress Controller side. 
+> 
+> After an Application Gateway is being linked to Application Gateway Ingress Controller (AGIC), all configuration will be synced and controlled by Ingress Controller. If you are trying to manually configure Application Gateway directly, it will be overwritten by Ingress Controller later. If there is a need to change settings on Application Gateway, you should change the relative settings, like annotations, from Application Ingress Ingress Controller.
```

Comment:
In addition to the wording change, can you please move this down to the "DevOps considerations" section of this article?

Reply:
@ckittel Thank you for your great suggestion. I have made relative change in the latest commit.